core/config/project_settings.cpp

```diff
@@ -46,6 +46,8 @@
 #include "core/version.h"
 
 #ifdef TOOLS_ENABLED
+#include "editor/editor_property_name_processor.h"
```

Comment:
Editor include in core 🙃

Reply:
Ah yeah, that's no good. Well, I was told to add this here... my original implementation when I opened the proposal was to add a new function to each module that returns a friendly name.

Reply:
Maybe you could apply it in Project Manager when displaying the features?